adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotationConverter.scala

```diff
@@ -21,11 +21,7 @@ import org.apache.avro.Schema
 import org.apache.avro.specific.SpecificRecord
 import htsjdk.variant.variantcontext.VariantContext
 import htsjdk.variant.vcf._
-import org.bdgenomics.formats.avro.{
-  DatabaseVariantAnnotation,
-  Genotype,
-  VariantCallingAnnotations
-}
+import org.bdgenomics.formats.avro.{ Genotype, GenotypeAnnotation, VariantAnnotation }
```

Comment:
How long is this line? I generally break the imports into multiple lines if I'm importing more than 4 things, or if the import line is >80 characters.